core/include/c_api/tiledb.h

```diff
@@ -190,6 +190,43 @@ typedef enum {
 #undef TILEDB_VFS_MODE_ENUM
 } tiledb_vfs_mode_t;
 
+/* ****************************** */
```

Comment:
This section appears to duplicate the previous constants section, unless there is some difference I am not seeing.

Reply:
That was probably my bad while rebasing. I will fix it. Thanks.